lib/driver/pg.js

```diff
@@ -255,6 +255,26 @@ var PgDriver = Base.extend({
         }]);
     },
 
+    /**
+    * Queries the migrations table
+    *
+    * @param callback
+    */
+    allLoadedMigrations: function(callback) {
+      var sql = 'SELECT * FROM ' + global.migrationTable + ' ORDER BY run_on DESC, name DESC';
+      return this.connection.query.apply(this.connection, sql);
+    },
+
+    /**
+     * Deletes a migration
+     * 
+     * @param migrationName   - The name of the migration to be deleted
+     */
+    deleteMigration: function(migrationName) {
+      var sql = 'DELETE FROM ' + global.migrationTable + ' WHERE name = ?';
```

Comment:
Like here, the correct quoting for pg is missing which is ", for mysql it is `. But you don't have to change this now. I will do this.